bin/configs/python-nextgen-aiohttp.yaml

```diff
@@ -5,3 +5,4 @@ templateDir: modules/openapi-generator/src/main/resources/python-nextgen
 library: asyncio
 additionalProperties:
   packageName: petstore_api
+  floatStrictType: false
```

Comment:
Is this feature only for asyncio library?

Reply:
OK, I've got it. It's for petstore examples and for aiohttp floatStrict is disabled.